tests/test_transforms/test_inference/conftest.py

```diff
@@ -6,6 +6,39 @@
 from etna.datasets import duplicate_data
 
 
+@pytest.fixture
+def ts_with_binary_exoc(random_seed) -> TSDataset:
```

Comment:
Try to use methods from `datasets.datasets_generation.py` for this purpose or create global fixture with binary column, which you will also use un tests for EventTransform